wrappers/ortools.rb

```diff
@@ -491,6 +492,20 @@ def build_rest(rest, day_index, job_load)
       build_detail(nil, rest, nil, day_index, job_load, nil, nil)
     end
 
+    def build_costs(costs)
+      cost_hash = {
+         fixed: costs&.fixed || 0,
+         time: costs && (costs.time + costs.time_fake + costs.time_without_wait) || 0,
+         distance: costs && (costs.distance + costs.distance_fake) || 0,
+         value: costs&.value || 0,
+         lateness: costs&.lateness || 0,
+         overload: costs&.overload || 0
+      }
+      total = cost_hash.values.reduce(&:+)
+      cost_hash[:total] = total
```

Comment:
Didn't tested but this should work right?
```suggestion
      cost_hash[:total] = cost_hash.values.reduce(&:+)
```